### ISSUE TITLE
Improve documentation about gnupg2 verification of certbot-auto

### ISSUE THREAD

**Issue by nulladmin** [certbot/certbot#5718]
This is a documentation issue regarding the verification steps for checking the signature integrity of the certbot-auto script. 

In the https://github.com/certbot/certbot/blob/master/docs/install.rst file there is a Hint where the following instructions are given for verifying the integrity of the certbot-auto script:

```
user@server:~$ wget -N https://dl.eff.org/certbot-auto.asc
user@server:~$ gpg2 --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
user@server:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.asc certbot-auto
```

If the user does not have gpg2 installed the instructions will fail, which is fine, we can assume that the user can figure out how to install gpg2 (e.g. `apt-get install gnupg2` tested on Ubuntu 14.04). However after installing gpg2 the second line will fail as follows:

```
user@server:~$ gpg2 --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
gpg: directory `/user/.gnupg' created
gpg: new configuration file `/user/.gnupg/gpg.conf' created
gpg: WARNING: options in `/user/.gnupg/gpg.conf' are not yet active during this run
gpg: keyring `/user/.gnupg/secring.gpg' created
gpg: keyring `/user/.gnupg/pubring.gpg' created
gpg: no keyserver known (use option --keyserver)
gpg: keyserver receive failed: Syntax error in URI
```

It would be useful to add an instruction for setting an appropriate key server to prevent confusion. For example (following the official gnupg FAQ https://www.gnupg.org/faq/gnupg-faq.html#define_keyserver)

```
user@server:~$ gpg2 --keyserver pool.sks-keyservers.net --recv-key A2CFB51FA275A7286234E7B24D17C995CD9775F2
```

Adding an example of a successful verification would also be helpful, e.g.

```
user@server:~$ gpg2 --trusted-key 4D17C995CD9775F2 --verify certbot-auto.asc /usr/local/sbin/certbot-auto
gpg: Signature made Wed 07 Mar 2018 06:23:57 PM UTC using RSA key ID CD9775F2
gpg: key CD9775F2 marked as ultimately trusted
gpg: checking the trustdb
gpg: 3 marginal(s) needed, 1 complete(s) needed, PGP trust model
gpg: depth: 0  valid:   1  signed:   0  trust: 0-, 0q, 0n, 0m, 0f, 1u
gpg: next trustdb check due at 2027-11-22
gpg: Good signature from "Let's Encrypt Client Team <letsencrypt-client@eff.org>"
```

This is admittedly a minor issue, but extra clarity when dealing with trust and verification is always useful.

**Comment by ohemorange** [comment]
This is a great idea, thanks! Would you like to pass on updating the documentation? Feel free to submit a patch against https://github.com/certbot/certbot/blob/master/docs/install.rst.

**Comment by R3DDY97** [comment]
Hi @bmw 
Is this in progress 
if not can i update the gnupg2 verification in documentation

**Comment by bmw** [comment]
@R3DDY97, submitting a PR to update this documentation would be great!

**Comment by R3DDY97** [comment]
Hi @bmw

**Comment by bmw** [comment]
Thanks for the PR @R3DDY97.

This might be a strange question, but are you at PyCon?

**Comment by R3DDY97** [comment]
oh
No

**Comment by akomm** [comment]
What would be really helpful is to simply start on the basics, like providing key(s) which can be used for verification AND the servers on which you have published them somewhere general, so at least when I google for it like "certbot gpg key" I actually find it.

Adding info on how to use it is a nice addition on top of this. Starting in the middle of a topic incomplete is rather confusing, especially for newbies, than helpful. Then better just give the base info required and link to outbound docs how to use those. My opinion.

**Comment by bmw** [comment]
I think this was resolved in #5981. If there are other changes someone would like to see, please submit a PR!
